Assignment.md

```diff
@@ -9,34 +9,70 @@
 1. Write a `for` loop that will iterate from `0` until `10`.
 1. For each iteration of the `for` loop, it will multiply the number by `9` and log the result (e.g. `"2 * 9 = 18"`).
 
+console.log("starting loop ...");
 **Bonus:**
 
 - Use a nested for loop to show the tables for every multiplier from 1 to 10 (100 results total)
 
-```js
+````js
 // Your Answer
-```
+`console.log("starting loop ...");
+
+for (let i = 0; i < 11; i++) {
+console.log(i * 9);
+}
+console.log("Ending loop ...");``
+
+
 
 ### 2: The Grade Assigner
 
 Check the results for every value from 60 to 100 - so your log should show "For 89, you got a B. For 90, you got an A.", etc.
 
 ```js
 // Your Answer
-```
+for (let i = 60; i < 101; i++) {
+  console.log(i);
```

Comment:
You can comment out this type of line after testing as it's not really part of the expected end result.